xarray/core/variable.py

```diff
@@ -1058,6 +1066,8 @@ def chunk(
         if chunkmanager.is_chunked_array(data_old):
             data_chunked = chunkmanager.rechunk(data_old, chunks)
         else:
+            if is_extension_array_dtype(data_old):  # dask cannot handle pandas types
```

Comment:
I'd prefer raising an error and asking the user to convert this variable manually. They won't know that something has changed until they call .compute() many steps later and will be quite confused.

Reply:
Done